subsys/dfu/src/dfu_target_mcuboot.c

```diff
@@ -161,7 +161,11 @@ int dfu_target_mcuboot_init(size_t file_size, dfu_target_callback_t cb)
 
 int dfu_target_mcuboot_offset_get(size_t *out)
 {
+#if !defined(CONFIG_DFU_TARGET_ALWAYS_RESTART)
```

Comment:
ifndef? I think this is more used elsewhere, but only if you feel like it :)

Reply:
Sure I just found `!defined` was used quite often in the `Zephyr` code base but if `#ifndef` is fine I'll start using that from now on :dango: